docs/src/main/asciidoc/datasource.adoc

```diff
@@ -68,17 +67,22 @@ quarkus.datasource.password=<your password>
 quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/hibernate_orm_test
 quarkus.datasource.jdbc.max-size=16
 ----
-<1> If you only have a single JDBC extension, or you are running tests and only have a single test scoped JDBC extension installed then this is
-optional. If there is only one possible extension we assume this is the correct one, and if a driver has been added with test scope then
-we assume that this should be used in testing.
+<1> If you only have a single JDBC extension or are running tests and only have a single test-scoped JDBC extension added, then this is optional.
```

Comment:
This whole aspect of single test-scoped JDBC seems confusing wording. Could we simplify this to point out that <1> is only required if there's more than one database extension on classpath? (And I don't think it's limited to JDBC).

Reply:
Will apply, thank you!